doc/source/js_integration.rst

```diff
@@ -0,0 +1,102 @@
+.. _js_integration:
+
+.. index:: js_integration
+
+Integrating Emperor's JavaScript API with other applications
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Emperor provides a rich API to manipulate the application live from the
+browser. The majority of this functionality is provided by attribute (Color,
+Visibility, Opacity, etc), or if needed the underlying `THREE` objects can
+be accessed as needed. For more information, the documentation for the
+`EmperorController` and other JavaScript classes can be accessed `here
+<../jsdoc/index.html>`_.
+
+
+In general, users won't need to do low-level manipulation of visual aspects but
+instead would like to subscribe to specific events and act as needed. For these
+cases, Emperor provides access to the following events:
+
+- When a sample is clicked.
```

Comment:
perhaps worth adding the name of the call/event or at least link them to their section?